.storybook/preview.js

```diff
@@ -0,0 +1,19 @@
+import { ThemeProvider } from '@material-ui/core/styles'
+import theme from 'src/utils/theme'
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
+
+export const decorators = [
```

Comment:
in general, storybook has a paradigm where it looks for specifically named exports in .stories files and in the .storybook/ folder.  Docs go over each of the named exports